lib/rucio/transfertool/fts3.py

```diff
@@ -174,8 +184,12 @@ def submit(self, files, job_params, timeout=None):
                                         headers={'Content-Type': 'application/json'},
                                         timeout=timeout)
             record_timer('transfertool.fts3.submit_transfer.%s' % self.__extract_host(self.external_host), (time.time() - start_time) * 1000 / len(files))
-        except:
-            logging.warn('Could not submit transfer to %s - %s', self.external_host, str(traceback.format_exc()))
+        except ReadTimeout as error:
+            raise TransferToolTimeout(error)
+        except JSONDecodeError as error:
+            raise TransferToolWrongAnswer(error)
+        except Exception as error:
+            logging.warn('Could not submit transfer to %s - %s' % (self.external_host, str(error)))
```

Comment:
same `continue` question

Reply:
That's the old behaviour, so there shouldn't be any problem